docker-alfresco/test/docker-alfresco-all-amps-test/test/docker-compose.yml

```diff
@@ -32,12 +31,33 @@ services:
                 -Djodconverter.url=http://libreoffice:8090/
                 -Dimg.url=http://imagemagick:8090/
                 -Dtika.url=http://tika:8090/
+                -Dalfresco.host=localhost
+                -Dalfresco.port.ssl=80
+                -Dalfresco.protocol=http
+                -Dalfresco.port=8082
+                -Dimap.server.enabled=true
+                -Dimap.server.port=1143
+                -Dimap.server.imaps.port=1143
+                -Dimap.server.host=0.0.0.0
+                -Dmail.protocol=smtp
+                -Dmail.port=465
+                -Dmail.host=smtp.gmail.com
+                -Demail.inbound.unknownUser=anonymous
+                -Demail.inbound.unknownUser=admin@alfresco.com
+                -Demail.inbound.enabled=true
+                -Demail.server.enabled=true
+                -Demail.server.port=1126
+                -Dsystem.usages.enabled=true
+                -Dcsrf.filter.enabled=false
                 "
         ports:
             - 8082:8080
             - 8000:8000
-            - 445:445
-            - 143:143
+            - 446:445
+            - 1143:1143
+            - 1126:1126
+            - 1125:1125
```

Comment:
why do we need this port exposed ?

Reply:
These ports are required for imap and smtp configuration. 1125 is used in one tests only and it needs to be exposed.

Reply:
that is what i suspected. thanks